core-web/libs/dotcms-scss/angular/dotcms-theme/components/_button.scss

```diff
@@ -2,708 +2,278 @@
+.p-button:not(.p-button-icon-only) {
+    gap: $spacing-1;
+    padding: 0 $spacing-3;
+    height: $spacing-6;
+    border-radius: 0.375rem;
+    border: none;
     color: $white;
     font-size: $font-size-md;
-    font-weight: $font-weight-semi-bold;
-    line-height: normal;
-    padding: $spacing-2 $spacing-3;
-    transition: background-color 0.2s, border-color 0.2s, color 0.2s, box-shadow 0.2s,
-        background-size 0.2s cubic-bezier(0.64, 0.09, 0.08, 1);
-    text-decoration: none;
-}
+    line-height: 140%;
+    text-transform: capitalize;
 
-a.p-button:hover,
-.p-button:enabled:hover,
-.p-button:enabled:active,
-.p-button:enabled:focus {
-    background: $color-palette-primary-600;
-    border-color: transparent;
-}
-
-.p-button.p-button-text {
-    border-color: transparent;
-    background-color: transparent;
-    color: $color-palette-gray-900;
-    overflow: hidden;
-    max-width: 100%;
-    border-radius: $border-radius-md;
+    .p-button-label {
+        color: $white;
+        font-size: $font-size-md;
+        line-height: 140%;
+        text-transform: capitalize;
+    }
 
+    .p-button-icon,
     .pi {
-        color: $color-palette-primary;
+        color: $white;
+        font-size: $font-size-lg;
+        line-height: 140%;
     }
 
-    &:disabled {
-        color: $color-palette-gray-500;
+    &.p-button-lg {
+        height: $spacing-7;
+        border-radius: $border-radius-md;
+        font-size: $font-size-lg;
 
+        .p-button-label {
+            font-size: $font-size-lg;
+        }
+
+        .p-button-icon,
         .pi {
-            color: $color-palette-gray-500;
+            font-size: $font-size-xl;
         }
     }
 
-    &:enabled:hover,
-    &:enabled:active,
-    &:enabled:focus {
-        color: $color-palette-gray-900;
-        box-shadow: none;
-    }
+    &.p-button-sm {
+        height: $spacing-5;
+        gap: 0.25rem;
+        padding: 0 $spacing-1;
+        border-radius: 0.25rem;
+        font-size: $font-size-sm;
 
-    &:enabled:hover {
-        background: $color-palette-primary-100;
-    }
+        .p-button-label {
+            font-size: $font-size-sm;
+        }
 
-    &:enabled:focus {
-        background: white;
-        box-shadow: $md-shadow-1;
+        .p-button-icon,
+        .pi {
+            font-size: $font-size-md;
+        }
     }
+}
 
-    &:enabled:active {
-        background: $color-palette-primary-200;
-    }
+// Severity colors for main basic button
+.p-button:enabled {
+    background-color: $color-palette-primary;
 
-    .p-button-label {
-        @include truncate-text;
-        text-transform: none;
-        font-weight: $font-weight-regular-bold;
+    &:hover {
+        background-color: $color-palette-primary-600;
+    }
+    &:active {
+        background-color: $color-palette-primary-700;
+    }
+    &:focus {
+        background-color: $color-palette-primary;
+        box-shadow: 0px 0px 0px 2.8px rgba(18, 67, 227, 0.2); // Ask what should we do
```

Comment:
All the focus will be the same not only for button but for other components, should be we create something to share this? maybe a mixin or a var? also we have to use opacity colors here, we have variables for this.

Reply:
`mixin field-focus()`